libpldmresponder/platform_config.hpp

```diff
@@ -46,6 +46,17 @@ class Handler : public CmdHandler
     void registerSystemTypeCallback(SystemTypeCallback callback);
 
   private:
+    /** @brief Interface to get the first available directory
+     *         available from the received list
```

Comment:
This "available" is redundant here and you could remove it

Reply:
corrected as suggestion.